pkg/virt-api/rest/vnc.go

```diff
@@ -29,11 +30,23 @@ func (app *SubresourceAPIApp) VNCRequestHandler(request *restful.Request, respon
 
 	defer apimetrics.SetVMILastConnectionTimestamp(request.PathParameter("namespace"), request.PathParameter("name"))
 
+	// Default is false as it should not drop an active connection
+	forceParam := false
+
+	// Check the request as QueryParameter assumes them to exist
+	if request.Request != nil && request.Request.URL != nil {
+		val, err := strconv.ParseBool(request.QueryParameter("force"))
+		if err != nil {
+			log.DefaultLogger().Reason(err).Warning("VNC's query parameter force")
```

Comment:
would it make sense to rephrase it to something like: `Failed to parse VNC's query parameter force`